packages/portal-proto/src/features/user-flow/workflow/registeredApps.tsx

```diff
@@ -228,6 +228,18 @@ export const REGISTERED_APPS = [
     noDataTooltip:
       "Current cohort does not have gene expression data available for visualization.",
   },
+  {
+    name: "Cohort Level MAF",
+    // TODO: replace with a MAF icon
+    icon: <BAMSlicingDownloadIcon aria-hidden="true" />,
```

Comment:
@wteouchicago Please generate an icon for this card, thanks. I've used another icon only to prototype.